pcdsdevices/analog_signals.py

```diff
@@ -14,10 +14,10 @@ class Acromag(Device, BaseInterface):
 
     Parameters:
     -----------
-    prefix : str
+    prefix : ``str``
```

Comment:
Backticks around builtin types is a bit weird. We generally follow the numpy docstring standard:
https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard

Reply:
Ahh 👌 I'll remove them

Reply:
The only times I see backticks mentioned in that guide are for variables names and equations. Should I remove our other usages too?

Reply:
It's not as clear cut in other scenarios, in my opinion. That jumps into sphinx territory from numpy docstring territory, and I'm not as much of a sphinx expert as I should be (so anyone feel free to correct me if I'm wrong).

Looking at the attenuator one, I see:
```
``Component``
```

Should really be:
```
:class:`ophyd.Component`
```

That one could probably be forgiven as it's obvious to us (PCDS).

References to arguments can be in backticks, such as:
```
``argument_name`` in :func:`function_name`.
```

Method name references should be like:

```
:meth:`class.method`
```

More here:
http://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#the-python-domain
Note that we usually strip off the domain `:py` at the beginning by convention.

Reply:
Got it 👍
How about for values like ``True`` or ``filter1`` or ``-1``?
I didn't see any explicit mention about a convention for them in either sphinx or numpy docs although in some spots. It looks like they are not backticked but I can imagine cases when you'd want them to be.

Reply:
Never mind, using quotes around strings handles those situations so I guess values should never be backticked.

Reply:
I forget, is there a `:keyword:` option? `:keyword:True` would be technically correct if so. I don't see a big problem with back-ticked `True` though, really.

I think it's fine to reference argument names by backticks or values you want to stand out.

Aside: sorry if I'm dragging down the PR with these stylistic things over functional things. Though maybe some good can come of it with one of us making a "docstring / sphinx style guide" of sorts on confluence...

Reply:
After this PR goes through, I'l try writing one up

Reply:
That'd be really great!